docs/OESDLBugBar.md

```diff
@@ -0,0 +1,86 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security. 
```

Comment:
Process wise, what are you proposing using the "severity" for?  E.g., in a triage meeting, what would we do differently for "important" vs "critical" bugs?

Reply:
The way I see it as the urgency in fixing them and how they are disclosed. Important and critical bugs needs to be fixed for all the supported version, while the moderate can be taken to the next release (it can also be considered for all the supported version). Does this answer your question?